tests/unit/src/Internal/Service/PaymentProcessingServiceTest.php

```diff
@@ -92,8 +96,6 @@ public function test_process_payment_happy_path() {
 	 * Test the basic happy path of processing a payment.
 	 */
 	public function test_process_payment_happy_path_without_mock_builder() {
-		$sut = new PaymentProcessingService( $this->mock_state_factory, $this->mock_legacy_proxy );
```

Comment:
This is the one method that should indeed use the `PaymentProcessingService` directly. Now coverage has gone down for the file, because `create_payment_context` is not covered with tests.

Reply:
Addressed here. https://github.com/Automattic/woocommerce-payments/pull/7471/commits/2c165178f28b4e6a39827c60fd9e55320d218bff